ivy/func_wrapper.py

```diff
@@ -596,6 +597,8 @@ def new_fn(*args, out=None, **kwargs):
         """
         if out is None:
             return fn(*args, **kwargs)
+        if handle_out_in_ivy:
```

Comment:
I don't think there would be a difference between `fn(*args, **kwargs)` and `fn(*args, out=out, **kwargs)` when `out` is `None`, let's just club these to make them more compact like,
```
if handle_out_in_ivy or out is None:
    return fn(*args, out=out, **kwargs)
```